openvdb/openvdb/python/pyGrid.h

```diff
@@ -1395,8 +1395,16 @@ meshToLevelSet(py::object pointsObj, py::object trianglesObj, py::object quadsOb
     return tools::meshToLevelSet<GridType>(*xform, points, triangles, quads, halfWidth);
 }
 
+template<typename GridType,
+    typename std::enable_if<!std::is_scalar<typename GridType::ValueType>::value>::type* = nullptr>
+inline py::object
+volumeToQuadMesh(const GridType&, py::object)
+{
+    OPENVDB_THROW(TypeError, "volume to mesh conversion is supported only for scalar grids");
+}
```

Comment:
I wonder if I should re-introduce this behaviour to `volumeToMesh`? Anyone that was previously instantiating this code with non-scalar grids but relying on it not being called will now encouter this compilation error. It's more correct but potentially disruptive? @danrbailey any thoughts?